lib/identity_doc_auth/mock/result_response_builder.rb

```diff
@@ -43,11 +45,28 @@ def call
       private
 
       def errors
-        error = parsed_data_from_uploaded_file&.dig('friendly_error')
-        if error.blank?
+        file_data = parsed_data_from_uploaded_file
+
+        if file_data.blank?
           {}
         else
-          { results: [error] }
+          image_metrics = file_data&.dig('image_metrics') || {}
+          failed = file_data&.dig('failed_alerts') || []
+          passed = file_data&.dig('passed_alerts') || []
+          liveness_result = file_data&.dig('liveness_result') || ''
```

Comment:
If we've already checked `file_data.blank?`, safe to omit the `&` ?

```suggestion
          image_metrics = file_data.dig('image_metrics') || {}
          failed = file_data.dig('failed_alerts') || []
          passed = file_data.dig('passed_alerts') || []
          liveness_result = file_data.dig('liveness_result') || ''
```

Reply:
probably...though I don't know what happens if the file_data object is not blank but is not the object type we're expecting...which I think in this code is possible. I'll check.